src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
```

Comment:
Give a more meaningful name, please 
```suggestion
  const calculator = {
```